.npmrc

```diff
@@ -0,0 +1,2 @@
+@fortawesome:registry=https://npm.fontawesome.com/
+//npm.fontawesome.com/:_authToken=75E4CDCD-60E5-495A-A554-DEE5002A3970
```

Comment:
This is a public repo and this token should not be committed. It should either be set through an environment variable or globally in the local npm configuration.

Consequently, this token needs to be rotated and replaced in all projects using it as this token is now compromised.